### ISSUE TITLE
prismic sm --setup failed on src directory

### ISSUE THREAD

**Issue by zipang** [prismicio/prismic-cli#70]
As you may know, Next.js now support the `src/ ` directory structure to heberge all the other sources and mainly : `pages/`
@see https://nextjs.org/docs/advanced-features/src-directory
In this situation (where the page folder is in fact located inside `src/pages/`) the `sm --setup` command will fail with the following message : 

```
 ERROR  ENOENT: no such file or directory, open 'pages/_app.js'                                               18:22:49

  at Object.openSync (fs.js:476:3)
  at Object.writeFileSync (fs.js:1467:35)
  at Object.write (/home/zipang/.config/yarn/global/node_modules/prismic-cli/bin/common/utils/files.js:26:20)
  at /home/zipang/.config/yarn/global/node_modules/prismic-cli/bin/commands/slicemachine/commands/setup.js:105:18
  at Array.forEach (<anonymous>)
  at createFiles (/home/zipang/.config/yarn/global/node_modules/prismic-cli/bin/commands/slicemachine/commands/setup.js:100:9)
  at _callee3$ (/home/zipang/.config/yarn/global/node_modules/prismic-cli/bin/commands/slicemachine/commands/setup.js:283:13)
  at tryCatch (/home/zipang/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:63:40)
  at Generator.invoke [as _invoke] (/home/zipang/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:293:22)
  at Generator.next (/home/zipang/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:118:21)
```

**Comment by emeraldsanto** [comment]
I'm also experiencing this, using `prismic-cli@3.8.4` and TypeScript.

**Comment by lihbr** [comment]
Hey thanks for the report! We'll have a look into it to support this feature 🙂

@hypervillain & @MarcMcIntosh as this have an impact on both the CLI and the builder I think

**Comment by hypervillain** [comment]
How would it impact the builder @lihbr?

**Comment by lihbr** [comment]
Resolving slices for the UI? Not sure perhaps it's already agnostic enough my bad ^^'

**Comment by MarcMcIntosh** [comment]
@zipang  thanks for pointing that out, it's been addressed in the commit/PR
fixed here: 5556464b1440afd6d8dd4248c8598a268d8a0d5b
